fanuc_r2000ib_support/package.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>fanuc_r2000ib_support</name>
+  <version>0.0.1</version>
+  <description>
+    <p>
+      ROS-Industrial support for the Fanuc R-2000iB (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Fanuc R-2000iB manipulators. This includes the /165F variant only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>R-2000iB/165F - "Conventional dress-out"</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information available
+      online at https://www.robots.com/robots/fanuc-r-2000ib-165f
+      All urdfs are based on the default motion
+      and joint velocity limits, unless noted otherwise (ie: no support for
+      high speed joints, extended / limited motion ranges or other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author email="simon.schmeisser@isys-vision.de">Simon Schmeisser (isys vision)</author>
+  <maintainer email="simon.schmeisser@isys-vision.de">Simon Schmeisser (isys vision)</maintainer>
```

Comment:
I would be ok with it, but are you going to maintain this? Otherwise, please don't list your name here.

Reply:
I think we should discuss increasing load-balancing, bus-factor and community involvement at some point and as such I would be willing to (co-)maintain some of those packages. However now it was just an oversight and I put the field back to its default value ;)